src/wp-includes/class-wp-phpmailer.php

```diff
@@ -27,37 +29,60 @@ public function __construct( $exceptions = false ) {
 	/**
 	 * Defines the error messages using WordPress' internationalization method.
 	 *
-	 * @return bool Always returns true in order to mimic PHPMailer's setLanguage() behavior
+	 * @since 6.8.0
+	 *
+	 * @return true Always returns true.
 	 */
 	public function SetLanguage( $langcode = 'en', $lang_path = '' ) {
```

Comment:
Can we just make more PHP 8+ compatible? 

Like, set the type to the function parameters.
`public function SetLanguage( string $langcode = 'en', string $lang_path = '' ): <return-type> {`

Address on other places as well.

Reply:
Hi @hbhalodia, since SetLanguage overrides the base PHPMailer method
```public function setLanguage($langcode = 'en', $lang_path = '')```
Adding parameter or return types is causing a compatibility error because PHP requires the method to match the parent class exactly.

Reply:
Okay, In that case we can skip this.